clarification requested by Dan Maharry (#1418)
oktadeveloper/okta-blog: _source/_posts/2020-12-21-beginners-guide-to-jwt.adoc

NOTE: Timestamps are "seconds since the epoch" integer format. JWT libraries usually add up to a few minutes of leeway to these values to account for clock skew between systems.

When an Okta authorization server returns an access or ID token to a client, two claims are of particular note:

 - Scopes (`scp`) - A list of accessible data points about the user - name, groups, etc. - and the client's API access rights as that user.

>  - Audience (`aud`) - A list of parties the token should be sent to and parsed by. The audience is the Okta Authorization Server Issuer URI for access tokens requesting Okta API access. The audience is the customer's API URI for access tokens requesting customer API access. For ID tokens, it's the client ID. Working with Okta, only one server is being targeted, so the list should only contain one item.

Reviewer: ```suggestion
 - Audience (`aud`) - A list of parties the token should be sent to and parsed by. In the access token, the audience is the Okta Authorization Server's Issuer URI requesting Okta API access or the customer's API URI requesting customer API access. In the id token, however, it's the client ID. Working with Okta, only one server is being targeted, so the list should only contain one item.
```
